app/charts/shared/interaction/tooltip-box.tsx

```diff
@@ -69,6 +69,30 @@ const usePosition = () => {
   return [box, handleRef] as const;
 };
 
+/**
+ * Responsible for folding the container size into a single value based on breakpoints.
+ * After accepting the container width, it accepts functions each breakpoint to return the generic value.
+ */
+const foldContainerSize =
+  <T,>(containerWidth: number) =>
```

Comment:
Super-nit: I think we can replace `<T,>` with` <T>`? Is there a reason for this or is it a typo? 😅